core/txpool/validation.go

```diff
@@ -54,6 +54,9 @@ type ValidationOptions struct {
 // This check is public to allow different transaction pools to check the basic
 // rules without duplicating code and running the risk of missed updates.
 func ValidateTransaction(tx *types.Transaction, head *types.Header, signer types.Signer, opts *ValidationOptions) error {
+	if !opts.Config.Is4844Enabled() && tx.Type() == types.BlobTxType {
+		return core.ErrTxTypeNotSupported
```

Comment:
I guess this is the part where blob tx is disabled and error is printed out without panic?

Reply:
Correct